help.php

```diff
@@ -1,7 +1,40 @@
 <?php
 // Include the header which contains the left menu, header
-include_once 'includes/header.php'; 
+include_once 'includes/header.php';
 ?>
+
+            <style>
+            .code-box {
+                position: relative;
+                background-color: #f0f0f0;
+                padding: 10px;
+                border-radius: 6px;
+                margin-bottom: 20px;
+            }
+            pre {
+                white-space: pre-wrap;
+                word-wrap: break-word;
+                font-family: Consolas, monospace;
+                margin: 0;
+            }
+            .copy-button {
+                position: absolute;
+                top: 5px;
+                right: 20px; /* Adjust as needed */
+                padding: 5px 10px;
+                background-color: #007bff;
+                color: white;
+                border: none;
+                border-radius: 4px;
+                cursor: pointer;
+            }
+            .copy-button:hover {
+                background-color: #0056b3;
+            }
+            .copy-icon {
+                margin-left: 5px;
+            }
+            </style>

```

Comment:
Can we put this in a css file instead?

Reply:
You are right. I forgot to move it ;) Thanks